src/pubsub/mod.rs

```diff
@@ -226,7 +226,8 @@ impl std::fmt::Display for Topics {
 #[derive(Serialize)]
 struct ITopicSubscribeData<'a> {
     topics: &'a [String],
-    auth_token: &'a str,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    auth_token: Option<&'a str>,
```

Comment:
I just noticed for some topics you don't need the access-token at all (e.g. `video-playback`). A followup PR could update `listen_command` to allow this. For now, passing an empty string to the function works as well.